### ISSUE TITLE
FEATURE - Support VoiceOver screen reader

### ISSUE THREAD

**Issue by SuzGupta** [mikaelacaron/brain-marks#137]
## Describe the feature you want the app to have that it currently doesn't
Let's add VoiceOver screen reader support as we start to think through how to improve accessibility for this app.

## Describe alternatives you've considered
Really, the main alternative is waiting and adding this support later. Right now, folks who rely on VoiceOver cannot utilize this app, so we are not supporting all possible users.

## Additional context
Here's the Apple doc on how to implement VoiceOver support: https://developer.apple.com/documentation/accessibility/supporting_voiceover_in_your_app

**Comment by ericjenkinson** [comment]
Hello. I would like to be assigned this request.

**Comment by ericjenkinson** [comment]
Hello Mikaela,

I am working on the VoiceOver and with SwiftUI all of the controls have Voice Over enabled. At this time the only change I have made is the not have voice over read the description of the background image as it does not aid in the navigation or description of the app. The new Category sheet is sort of distracting via VoiceOver as every button, label, and icon are read. I do not know if there is a benefit to VoiceOver user to have all of the icons read.

Are there any other items I should be looking at for this issue?

**Comment by mikaelacaron** [comment]
@ericjenkinson Thanks! Mainly this was an audit, because I wasn't sure what all was covered. I don't think we can do any voice over with the images in a tweet unless there's a way to fetch the alt text of a tweet

I would think that the new category sheet, it'd be useful to only read the icons when a person is hovering over them, as opposed to reading them all the second you see the view

What do you think?

**Comment by ericjenkinson** [comment]
Great. I will remove the reading of the icons as a set, but still read the icon when the user navigates through them.

**Comment by mikaelacaron** [comment]
@ericjenkinson  Perfect thanks! Let me know if you have any other questions!
